Documentation/GitHub/Index.rst

```diff
@@ -21,8 +21,16 @@ But, you may want to get an overview for all open issues for TYPO3 documentation
 The repositories of the official manuals are all included in the organization
 `TYPO3-Documentation <https://github.com/TYPO3-Documentation>`__.
 
+Pick an issue for a topic you are familiar with and try to fix it. Some of the
+issues address a problem, some are enhancements where new text needs to be
+written.
+
 Here are some links you can use to get a list of all issues:
 
+.. note::
+
+   GitLab will show a 404 page while you are not logged in following these links!
```

Comment:
This should be GitHub and not Gitlab. I will push another commit shortly for this.

The rest is a great improvement. Thanks :thumbsup: